src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        if (file.exists()) {
+            file.delete();
+        }
+        String[] result = countResult(readFromFile(fromFileName));
+        writeToFile(file,result);
+    }
+
+    public String[] readFromFile(String fileName) {
+        File file = new File(fileName);
+        List<String> arrayList = new ArrayList<>();
```

Comment:
rewrite using StringBuilder. You haven't studied List yet

Reply:
is it necessary to use StringBuilder instead of list?Because i`ve reviewed other approved works and there were no need to use StringBuilder...